stores/stripe.ts

```diff
@@ -1,13 +1,21 @@
-import { defineStore } from 'pinia'
+import { defineStore, storeToRefs } from 'pinia'
+import { useBookStoreApiStore } from './book-store-api'
 import { LIKE_CO_API } from '~/constant'
```

Comment:
Absolute path import come first
```suggestion
import { LIKE_CO_API } from '~/constant'

import { useBookStoreApiStore } from './book-store-api'
```

Reply:
My eslint doesn't allow me to do this 😅

[🎨 Update import path](https://github.com/likecoin/nft-book-press/pull/176/commits/e817919977412a22001bf406f19fe09c61b307d6)